### DISCUSSION TITLE
Multicast and Broadcast Service (MBS) support

### DISCUSSION THREAD

**Discussion by vamaglog** [srsran/srsRAN_Project#522] [Ideas & Suggestions]
Hi,
Are there any plans to support multicast/broadcast services over the 5G implementation? 
I am aware of the MBSFN implementation in LTE, but is it on the roadmap to also support MBS in srsRAN Project for 5G?
If yes, by when? And is there any git branch with work in progress? 
Thank you!

**Comment by yagoda** [comment]
Hi,

This is not in our immediate roadmap, we will update you if/when that changes.

Regards,
Justin
